Mathlib/Topology/Sheaves/SheafCondition/UniqueGluing.lean

```diff
@@ -319,8 +319,8 @@ theorem eq_of_locally_eq₂ {U₁ U₂ V : Opens X} (i₁ : U₁ ⟶ V) (i₂ :
     · refine' le_trans hcover _
       rw [sup_le_iff]
       constructor
-      · convert le_iSup (fun t : ULift Bool => if t.1 then U₁ else U₂) (ULift.up True)
-      · convert le_iSup (fun t : ULift Bool => if t.1 then U₁ else U₂) (ULift.up False)
+      · convert le_iSup (fun t : ULift Bool => if t.1 then U₁ else U₂) (ULift.up true)
```

Comment:
Should this whole proof use `bif` instead of `if`? If so that can wait for a later PR.

Reply:
I thought about that, but didn't have lean open so couldn't actually change any proofs.